source/containers.tex

```diff
@@ -6058,6 +6058,7 @@
 \libheaderref{array},
 \libheaderref{deque},
 \libheaderrefx{forward_list}{forward.list.syn},
+\libheaderrefx{hive}{hive.syn},
```

Comment:
Since "hive.syn" is regular (derived from "hive"), please use `\libheaderref`.

Reply:
Done